app/actions/NoCacheActionBuilder.scala

```diff
@@ -0,0 +1,17 @@
+package actions
+
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class NoCacheActionBuilder(
+  val parser: BodyParser[AnyContent],
+  val executionContext: ExecutionContext
+)
+    extends ActionBuilder[Request, AnyContent] {
+  implicit private val ec = executionContext
+
+  override def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]): Future[Result] =
+    block(request).map(_.withHeaders("Cache-Control" -> "no-cache, private", "Pragma" -> "no-cache", "Expires" -> "0"))
```

Comment:
You could use the existing constant: https://github.com/guardian/support-frontend/blob/b8d083706a57dbbd48eed6858887b68deb9917b1/app/actions/CacheControl.scala#L20

Reply:
👍